packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -120,9 +121,13 @@ const AccountTransferForm = observer(
             resetConverter,
         } = crypto_fiat_converter;
 
+        const is_migration_status_present =
+            selected_to.status === MT5AccountStatus.MIGRATED_WITH_POSITION ||
+            selected_to.status === MT5AccountStatus.MIGRATED_WITHOUT_POSITION;
+
         const [from_accounts, setFromAccounts] = React.useState({});
         const [to_accounts, setToAccounts] = React.useState({});
-        const [transfer_to_hint, setTransferToHint] = React.useState<string>();
+        const [transfer_to_hint, setTransferToHint] = React.useState<JSX.Element>();
```

Comment:
May i know why do we need `JSX.Element` in a state?

Reply:
`setTransferToHint` will be taking in `Localize` component later in code. so type needs to be mentioned here.